teams/tiup/membership.json

```diff
@@ -0,0 +1,23 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "AstroProfundis",
+        "lonng",
```

Comment:
umm, I think it's ok to move him to committers (

Reply:
updated